dat/missions/dvaered/dvaered-colonel-transport.lua

```diff
@@ -0,0 +1,56 @@
+--[[
+<?xml version='1.0' encoding='utf8'?>
+<mission name="Escort a Dvaered colonel to a system">
+ <unique />
+ <priority>1</priority>
+ <chance>24</chance>
+ <location>Bar</location>
+ <spob>Dvaer Prime</spob>
+</mission>
+--]]
+--[[
+
+   MISSION: ESCORT TO A PLANET
+   DESCRIPTION: SMALL MISSION WHERE YOU ESCORT AN ARSENAL TO A PLANET: WIP
+
+--]]
+
+local fmt = require "format"
+
+
+function create ()
+   mem.talked = false
+   misn.setNPC( _("A Dvaered colonel"), "devaered/dv_military_f6.webp", _("This soldier seems to be a colonel.") )
+end
+
+function accept ()
+   local text
+   else
+      text = fmt.f(_([[You approach the Dvaered, who seems to be a soldier, probably one of the colonel rank. Arriving at their table, you are greeted, "Hello! Could you escort my ship, an Arsenal, to {pnt} in the {sys} system? I'll give you {rwd} for it, but I can't tell you why. Well, what do you say?"]]),
+         {pnt=misplanet, sys=missys, rwd=reward_text})
+      mem.talked = true
+   end
+   if vntk.yesno( _("Escort Agreed"), text ) then
```

Comment:
I would recommend you use the `vn` library, and not `vntk`. In general, most modern missions use `vn` and display the NPC portrait. You can generate one automatically with the `vnimage` module, such as `vnimage.dvaeredMilitary()` for a random dvaered military character (it can be male or female looking).